### PR TITLE
Completed bigO and persistence

### PR DESCRIPTION
**Pull request by con-soul**

### CODE REVIEW

**Comment by con-soul** [comment]
I put my  persistence.js file in the _submissions/matheson-mckay folder, so I cp'd it over to the persistence folder

File: hw3/persistence/persistence.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// Write a function, persistence, that takes in a positive parameter `num`
```
Comment (patrik-drean) [inline hw3/persistence/persistence.js]: This file shouldn't be here.

File: hw3/_submissions/matheson-mckay/persistence.js [hunk 2]
```diff
@@ -1,6 +1,23 @@
+// Write a function, persistence, that takes in a positive parameter `num`
```
Comment (patrik-drean) [inline hw3/_submissions/matheson-mckay/persistence.js]: This moves the template file to your submission. We need this file to stay here unaltered for other students.

**Comment by con-soul** [comment]
K I think I finally got it this time. I think I meant to cp instead of move the 1-TEMPLATE.js file

**Comment by con-soul** [comment]
Got HW4 done

**Pull request merged**